docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -277,6 +283,80 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+## Sending Search Requests from Client to Server
```

Comment:
Please change to:

## Sending and Receiving Search Requests from Client to Server